playbooks/openshift-master/private/tasks/enable_bootstrap.yml

```diff
@@ -0,0 +1,22 @@
+---
+- when: openshift_master_bootstrap_enabled | default(false) | bool
```

Comment:
This boolean appears to be somewhat redundant.  Probably should remove for consistency; alternatively, remove the other bools.

Reply:
I can do that.